docs/gc.md

```diff
@@ -35,31 +35,34 @@ To prevent applications running out of memory, objects in the Java heap that are
 
-| GC Policy                                        | Heap Configuration              | Collectors                         |
-|--------------------------------------------------|---------------------------------|------------------------------------|
-| [`gencon     `](#gencon-policy-default) (default)| Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
-| [`balanced   `](#balanced-policy)                | Generational (multiple regions) | Incremental Generational           |
-| [`metronome  `](#metronome-policy)               | Segregated                      | Realtime                           |
-| [`optavgpause`](#optavgpause-policy)             | Flat                            | Concurrent Mark, Global            |
-| [`optthruput `](#optthruput-policy)              | Flat                            | Global                             |
-| [`nogc       `](#nogc-policy)                    | --                              | --                                 |
+| GC policy name (option name)                                           |  Heap configuration             | Collectors                         |
+|------------------------------------------------------------------------|---------------------------------|------------------------------------|
+| [Generational concurrent (`gencon`)](#gencon-policy-default) (default) | Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
+| [Balanced (`balanced`)](#balanced-policy)                              | Generational (multiple regions) | Incremental Generational           |
+| [Metronome (`metronome`)](#metronome-policy)                           | Segregated                      | Realtime                           |
+| [Optimize for pause time (`optavgpause`)](#optavgpause-policy)         | Flat                            | Concurrent Mark, Global            |
+| [Optimize for throughput (`optthruput`)](#optthruput-policy)           | Flat                            | Global                             |
+| [No garbage collection (`nogc`)](#nogc-policy)                         | Flat                            | --                                 |
 
-All OpenJ9 GC policies support compressed references on 64-bit platforms, which compresses heap pointers to 32 bits if the total heap size does not exceed 63 GB. Applications that require more heap space can forgo compressed pointers and select any heap size within the bounds imposed by the operating system and available system RAM.
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** All OpenJ9 GC policies support compressed references on 64-bit platforms, which compresses heap pointers to 32 bits if the total heap size does not exceed 63 GB. Applications that require more heap space can forgo compressed pointers and select any heap size within the bounds imposed by the operating system and available system RAM.
```

Comment:
63GB is neither theoretical nor practical accurate limit. We should either say _approximately_ 63GB or say 64GB (what would be at least theoretical upper bound). Either way, eventually we should point to somewhere else from more details - there are various interesting/relevant details about practical heap sizing limits and when we use 1, 2, 3 or 4 bit shifting during compress/decompress.

Reply:
Yes, 63GB is meaningless from any point (and even impossible for some platforms). With Mixed (Compressed Refs/Full 64 bit) build in place (coming soon) the switch is going to occur at 57GB. If we don't want to go to these details here we can mention 64GB as theoretical upper bound as Aleks suggested

Reply:
Thanks. Replacing with "....if the total heap size does not exceed the theoretical upper bound of 64 GB"